scripts/Configure.sh

```diff
@@ -32,17 +32,17 @@ while [[ $# -gt 0 ]]; do
       shift
       shift
       ;;
-    --completion)
+    --completionmodel)
```

Comment:
This impacts the docs on learn. Please submit a PR to fix that before this is merged.

Reply:
oh snap.  where do I submit a PR for learn docs?

Reply:
@matthewbolanos If you are talking about what's in https://github.com/MicrosoftDocs/semantic-kernel-pr/ I couldn't find references to the "completion" parameter in those documents and we should be good to go.

Are there other documents??

Reply:
This PR https://github.com/MicrosoftDocs/semantic-kernel-pr/pull/134 looks great :) Thanks @momuno. Let me know when this PR is merged in and then I'll do the same on the docs.